Marlin/src/pins/stm32f4/pins_FYSETC_CHEETAH_V20.h

```diff
@@ -44,14 +44,9 @@
 #endif
 
 //
-// Z Probe
+// Servos
 //
-#if ENABLED(BLTOUCH)
-  #error "You need to set jumper to 5v for Bltouch, then comment out this line to proceed."
```

Comment:
Why was this condition believed to be needed? Is the 5V jumper not actually needed? Where did the information about a 5V jumper come from originally?